docs(faq): update content for SEO optimization MTA-1762 (#417)
scaleway/docs-content: faq/apple-silicon-m1-as-a-service.mdx

---
meta:
  title: Apple silicon M1 as-a-service

>   description: Discover how to use M1 for your OS applications.

Reviewer: ```suggestion
  description: Discover how to use M1 for applications development.
```

>   description: Discover how to use M1 for applications development.

Author: ```suggestion
  description: Discover how to use M1 for applications' development.
```

>   description: Discover how to use M1 for applications' development.

Reviewer 2: `Discover how to use M1 for application development` would sound better I think?

content:
  h1: Apple silicon M1 as-a-service

>   paragraph: Discover how to use M1 for your OS applications.

Reviewer: ```suggestion
  paragraph: Discover how to use M1 for applications development.
```

>   paragraph: Discover how to use M1 for applications development.

Author: ```suggestion
  paragraph: Discover how to use M1 for applications' development.
```

>   paragraph: Discover how to use M1 for applications' development.

Author: ```suggestion
  paragraph: Discover how to use M1 for application development.
```
